### PR TITLE
Don't test wrapHandler when testing jsonHandler.

### PR DESCRIPTION
**Pull request by abhinav**

### CODE REVIEW

File: encoding/json/inbound_test.go [hunk 1]
```diff
@@ -25,17 +26,19 @@ type simpleResponse struct {
 }
 
 func TestHandleStructSuccess(t *testing.T) {
-	h := wrapHandler(
-		"foo",
-		func(_ context.Context, _ yarpc.Meta, r *simpleRequest) (*simpleResponse, yarpc.Meta, error) {
-			assert.Equal(t, "foo", r.Name)
-			assert.Equal(t, map[string]int32{"bar": 42}, r.Attributes)
+	f := func(_ context.Context, _ yarpc.Meta, r *simpleRequest) (*simpleResponse, yarpc.Meta, error) {
```
Comment (breerly) [inline encoding/json/inbound_test.go]: Probably should use `h` for handler instead of `f`.

**Comment by breerly** [comment]
lgtm. :ship:

**Comment by prashantv** [comment]
lgtm

File: encoding/json/inbound_test.go (hunk 1 above)
Comment (abhinav) [inline encoding/json/inbound_test.go]: Fixed

**Pull request merged**